### DISCUSSION TITLE
No Connection since update

### DISCUSSION THREAD

**Discussion by TomFeix** [StephanJoubert/home_assistant_solarman#560] [Q&A] [answered]
Error in Protokoll
Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: custom_components.solarman.solarman
Quelle: custom_components/solarman/solarman.py:88
Integration: Solarman (Dokumentation, Probleme)
Erstmals aufgetreten: 18:55:02 (320 Vorkommnisse)
Zuletzt protokolliert: 20:15:03

Querying [3 - 89] failed with exception [OSError: [Errno 30] Read-only file system]
Querying [3 - 89] failed with exception [NoSocketAvailableError: No socket available]

**Comment by danielmessem** [comment]
Same for me, solorman not connecting after updating to homeassistant core 2024.4.3
Rolled back to 2024.4.2 and it works just fine :)

**Reply by TomFeix** [reply, marked as answer]
Thanks - works fine

**Comment by Tobias1122334** [comment]
Hello,

will there be an update or work-around to get the integration running under HA 2024.4.3?

**Comment by TomFeix** [comment]
Hello,

will there be an update or work-around to get the integration running under HA 2024.5.1?
